Source/RunActivity/Viewer3D/Signals.cs

```diff
@@ -413,6 +413,7 @@ void renderEffect(Material material)
             internal void Mark()
             {
                 SignalTypeData.Material.Mark();
+                SignalTypeData.GlowMaterial?.Mark();
```

Comment:
Is there any reason to guard against null here? The glow material isn't loaded from the route data.

Reply:
In the constructor below this line there is an if-else condition, and the GlowMaterial isn't set in both cases.

Reply:
Gotcha